packages/ui/src/hooks/useMultisigsBySignatoriesOrWatchedSubscription.tsx

```diff
@@ -106,5 +106,11 @@ export const useMultisigsBySignatoriesOrWatchedSubscription = ({
   // console.log('subscription data', data)
   //   return <div>Data: {JSON.stringify(data?.multisigCalls)}</div>;
 
-  return { data, isLoading: hasSomethingToQuery && isLoading, error, refetch }
+  return {
+    data,
+    isLoading: hasSomethingToQuery && isLoading,
+    isFetchingMultisigs: isFetching,
```

Comment:
I was wondering why not using `isLoading`? From [the docs for isLoading](https://tanstack.com/query/v5/docs/react/reference/useQuery):
>  Is true whenever the first fetch for a query is in-flight
    Is the same as isFetching && isPending

so I think it's not needed to add is, we can use `isLoading`.